CMS/static/scss/global.scss

```diff
@@ -207,7 +207,7 @@ a[target="_blank"] {
     margin-top: 1em;
 
     img {
-      height: 200px;
+      width: 100%;
```

Comment:
sorry about this - I realise why I had a 200px hight to start with - it was to make sure the image looked totally distinct from the resource cards (which are also fixed height)

could you do it so that it keeps 200px as a max-height?

This may help if needed...

https://stackoverflow.com/questions/12991351/css-force-image-resize-and-keep-aspect-ratio

Reply:
OK - set max-height to 200px and max-width to 100%, that seems to cover both bases.